libs/application/templates/children-residence-change/src/fields/Interview/index.tsx

```diff
@@ -7,9 +7,10 @@ import { CRCFieldBaseProps } from '../../types'
 import { DescriptionText } from '../components'
 
 type ValidAnswers = 'yes' | 'no'
+type FieldIds = 'interview' | 'interviewParentB'
 
 const Interview = ({ field, application, error }: CRCFieldBaseProps) => {
-  const currentAnswer = application.answers.interview
+  const currentAnswer = application.answers[field.id as FieldIds]
```

Comment:
Did this instead of doing:

```
const currentAnswer = application.status === 'draft' ? application.answers.interview : application.answers.interviewParentB
```